src/services/routingSlip.services.ts

```diff
@@ -52,6 +52,24 @@ export default class RoutingSlip {
     )
   }
 
+  public static async updateRoutingSlipRefundStatus (
+    code: string,
+    routingSlipNumber: string
+  ): Promise<AxiosResponse> {
```

Comment:
What type is the response? if it's simple, wouldn't recommend an interface.. if it's more complex and you use it, would recommend it

Reply:
it's same as updating status before.
<img width="678" alt="image" src="https://github.com/user-attachments/assets/17cc5d71-56f3-42eb-85a0-2075ce55ae41">